assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -101,10 +104,10 @@ che.workspace.hosts=NULL
 # This is the API endpoint of the workspace master running within the core Che server.
 # This tells the workspace agent how to connect back to the Che server.
 # che-host is a hostname entry added to /etc/hosts of the workspace by the Che server.
-che.workspace.che_server_endpoint=http://che-host:${CHE_PORT}/wsmaster/api
+che.infra.docker.master_api_endpoint=http://che-host:${CHE_PORT}/wsmaster/api
```

Comment:
Let's move these docker infra properties in [Che docker infrastructure parameters](https://github.com/eclipse/che/blob/c4346a9663f06b269dedef82b52a5a4c8ed80bfa/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties#L282) section